graphql/mutation_resolver.go

```diff
@@ -175,37 +174,32 @@ func (r *mutationResolver) CreateDistro(ctx context.Context, opts CreateDistroIn
 	}, nil
 }
 
-// SaveDistroSection is the resolver for the saveDistroSection field.
+// SaveDistroSection is the resolver for the saveDistroSection field. The entire distro object is provided as input (not just the updated fields) in order to validate all distro settings.
```

Comment:
nit: can we make this two lines since it's a bit long?

Reply:
Sadly gqlgen seems to impose a limit of one line of comments before a resolver. When I use two lines it tries to force this:
```go
// SaveDistroSection is the resolver for the saveDistroSection field. The entire distro object is provided as input
(not just the updated fields) in order to validate all distro settings.
	func (r *mutationResolver) SaveDistroSection(ctx context.Context, opts SaveDistroInput) ( *SaveDistroSectionPayload,  error){
```